web/src/context/SampleManifest.tsx

```diff
@@ -0,0 +1,68 @@
+import { createContext, useContext, useEffect, useState } from "react";
+import { BASE } from "../site";
+
+interface Manifest {
+  models: string[];
+  languages: string[];
+  templates: string[];
+  postprocessors: string[];
+}
+
+const emptyManifest: Manifest = {
```

Comment:
there is no point to this since useSampleManifest() returns `Manifest | undefined` - we can just use undefined.